<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>KoPub돋움체 Bold</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Exel to CSV</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_csvpath">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_csvfile">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>csv 파일</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_csv_search">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>찾기</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_savepath">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>파일 저장 경로</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_savepath">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_savefile_search">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>찾기</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_rmv_header">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>첫 헤더 지우기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_convert">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>변환하기</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
